puppet/chassis.rb

```diff
@@ -229,4 +230,103 @@ def self.make_relative(base, paths)
 			pathname.relative_path_from(base).to_s
 		end
 	end
+
+	def self.update_core
+		puts "\e[32mChecking for Chassis core updates...\e[0m"
```

Comment:
If there's no updates, can we clear this message line to avoid extra output noise? I think it'd be `puts "\r\e[K"`, but not certain.

Reply:
`puts "\033[A\r\e[K"`seemed to do the trick. `\033[A` moves the cursor up one line.